core/src/test/java/ai/timefold/solver/core/impl/testdata/domain/declarative/fsr/TestdataFSRAssertionConstraintProvider.java

```diff
@@ -0,0 +1,30 @@
+package ai.timefold.solver.core.impl.testdata.domain.declarative.fsr;
+
+import java.time.Duration;
+
+import ai.timefold.solver.core.api.score.buildin.hardsoft.HardSoftScore;
+import ai.timefold.solver.core.api.score.stream.Constraint;
+import ai.timefold.solver.core.api.score.stream.ConstraintFactory;
+import ai.timefold.solver.core.api.score.stream.ConstraintProvider;
+
+import org.jspecify.annotations.NonNull;
+
+public class TestdataFSRAssertionConstraintProvider implements ConstraintProvider {
```

Comment:
Is FSR being copy pasted into the solver test sources?
Or is it a subset?

Reply:
A **heavily** simplified subset.

Reply:
Please remove references to FSR. FSR has no meaning in the context of the solver.